### PR TITLE
Add method to sort forumTopics by last post

### PR DESCRIPTION
**Pull request by amberwebb**

### CODE REVIEW

**Comment by nathanielvelez** [comment]
:+1:

File: src/collections/forumTopics.coffee [hunk 1]
```diff
@@ -32,3 +32,14 @@ exports.deleteForumTopic = (forumTopic, callback) ->
       be provided'
 
   @deleteItem forumTopic, callback
+
+exports.getForumTopicPostsSort = ->
+  (itemA, itemB) ->
+    valueA = new Date(itemA.forumPosts.last().createdAt)
```
Comment (matthopson) [inline src/collections/forumTopics.coffee]: How does this hold up when there are no posts for a given topic?

File: src/collections/forumTopics.coffee [hunk 2]
```diff
@@ -32,3 +32,14 @@ exports.deleteForumTopic = (forumTopic, callback) ->
       be provided'
 
   @deleteItem forumTopic, callback
+
+exports.getForumTopicPostsSort = ->
```
Comment (matthopson) [inline src/collections/forumTopics.coffee]: I guess we don't care about being able to reverse the sort order?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # TeamSnap JavaScript SDK CHANGELOG
 
+### March 17, 2016 // Version 1.13.0
+- Adds `getForumTopicPostsSort` method.
```
Comment (matthopson) [inline CHANGELOG.md]: This method only sorts `forumTopic` items, not `posts`.

File: src/collections/forumTopics.coffee (hunk 1 above)
Comment (amberwebb) [inline src/collections/forumTopics.coffee]: There's always a forumPost. `forumTopic` is a container for `forumPost`. So when you create a topic, you are also creating a post.

File: src/collections/forumTopics.coffee (hunk 2 above)
Comment (amberwebb) [inline src/collections/forumTopics.coffee]: No, we don't. ;)

In nextjenn, if I specify `desc` in the sort, it reverses the order.

File: src/collections/forumTopics.coffee (hunk 1 above)
Comment (matthopson) [inline src/collections/forumTopics.coffee]: But it isn't necessarily loaded (for instance in the case of Onion).

Comment (amberwebb) [inline src/collections/forumTopics.coffee]: Do you mean forumPosts? Not following why forumPosts wouldn't be loaded if forumTopics are.

**Pull request closed without merging**